compiler/src/iree/compiler/Codegen/Dialect/GPU/Transforms/test/distribute_mma_to_lanes.mlir

```diff
@@ -31,3 +31,85 @@ module {
 //  CHECK-SAME:       kind = #iree_gpu.mma_layout<MFMA_F32_16x16x16_F16>
```

Comment:
Add tests for newly supported intrinsics, e.g.,
- `MFMA_F32_16x16x4_F32`
- `MFMA_F32_16x16x32_F8E4M3FNUZ`
- `MFMA_I32_32x32x16_I8`
- `WMMA_F16_16x16x16_F16`